accept/src/botReview.ts

```diff
@@ -2,10 +2,56 @@ import { getInput, info, warning } from '@actions/core'
 import { getOctokit } from '@actions/github'
 import { getPR, getShaFromBranch } from '../../lib/github'
 import { detectIssueKey } from './detectIssueKey'
+import { getEventData } from '../../lib/event'
 
 // id of exivity bot
 const EXIVITY_BOT = 53756225
 
+interface IDObject {
+  id: number
+}
+
+async function checkIfReady(
+  octokit: ReturnType<typeof getOctokit>,
+  ref: string,
+  repo: string
+): Promise<boolean> {
+  const event = process.env['GITHUB_EVENT_NAME']
+
+  if (event === 'push') return true
+
+  const eventData = await getEventData()
+
+  if (
+    event === 'check_run' &&
+    eventData.check_run?.check_suite.pull_requests.requested_reviewers.some(
+      (reviewer: IDObject) => reviewer.id === EXIVITY_BOT
+    )
+  ) {
+    const checkResult = await octokit.checks.listForRef({
+      owner: 'exivity',
+      repo,
+      ref,
+    })
+
+    return checkResult.data.check_runs?.every(
+      (check) => check.status === 'completed' && check.conclusion === 'success'
+    )
+  }
+
+  if (event === 'pull_request') {
```

Comment:
This needs to also check if the check suite for latest commit has completed with conclusion === success.